Readme Update

NBumgardner opened the pull request:
This pull request **polishes the main readme**. My fork has a [preview of the PR](https://github.com/NBumgardner/Floe/tree/readme-update).
- Rewords two sections (_How do I get set up?_ and _Who do I talk to?_) for flow and conciseness.
- Added my name to the contacts, with space below for newcomers.
- Adds links to contributors and demos (`SavedBinFiles` folder) for navigation.
- Adds backquotes around all code snippets,  commands, and folders; for clarity and consistency.
- Changes level 3 headings to level 2, since only two heading levels are used.

Minor, consistent punctuation fixes include:
- Italicizes version number.
- Bolds debug warning instead of using all-caps.
- Minor punctuation fixes (for periods and colons).

NBumgardner committed:
Reword readme.md Set Up, Add Demos Link

NBumgardner committed:
Add readme.me punctuation, italicize version no.

NBumgardner committed:
Change readme.md caps to bold

NBumgardner committed:
Rearranged readme.md Contacts Section, Added Myself

NBumgardner committed:
Linked Names in readme.md, added line break

NBumgardner committed:
Remove extra colon in readme.md heading

NBumgardner committed:
Reduce readme.md level 3 headings to level 2

NBumgardner committed:
Add bullet before @SamBumgardner for consistency

SamBumgardner commented on readme.md:
```diff
@@ -2,46 +2,51 @@
 
 Help Floe, the spirit of ice, bring winter to the land by freezing over lakes. There are a number of different enemies that stand in your path, however, and they aren't happy about Floe's intrusion on their territory. 
-Version 1.0
+*Version 1.0*
```
If you could move the version down by another line (so it's not just at the end of the previous sentence) that'd be great!

NBumgardner replied:
I'm on it.

SamBumgardner commented on readme.md:
```diff
@@ -2,46 +2,51 @@
 * Freeze all water tiles in the level to move on to the next.
 * Avoid all enemies.
-* DEBUG MODE: use the 'd' key to damage yourself, use 'n' to advance to the next level.
+* **Debug Mode:** Use the 'd' key to damage yourself, use 'n' to advance to the next level.
```
What do you think of putting 'd' and 'n' in backquotes, so they're more clearly related to keyboard keys?

Ex: 
- **Debug Mode:** Use the `d` key to damage yourself, use `n` to advance to the next level.

NBumgardner replied:
I'm on it.

SamBumgardner commented on readme.md:
```diff
@@ -2,46 +2,51 @@
+ 2. Open a command prompt in `/Floe/Floe/`.
+ 3. Run the command `lime test flash` to build and run the executable.
+
+* The executable is located in `/Floe/Floe/bin/flash/bin/`.
```
I don't know if I like having a bullet-point list immediately following a numbered list. What do you think of making these two points nested inside the above list?

NBumgardner replied:
Do you want the bullets to become nested letters, nested bullets, or a following paragraph? I'll try them myself too.

SamBumgardner reviewed (commented):
Overall, looks like a nice pull request. I left a few comments detailing some possible changes; please let me know if you think they're worthwhile.

NBumgardner committed:
Alter readme.md as requested on PR x2

Move version number to appear on a separate line.
Change key names from being single-quoted to back-quoted.

NBumgardner committed:
Convert 'Getting Started' bullets into sub-bullets

NBumgardner commented:
This current PR has all three changes requested by @SamBumgardner so far.

**Future Notes:**
- I **prefer** placing the **version number inline** within the first two header sections. It would keep the readme shorter, and I think most readers will skip the version number.
- The **version number** may need to be **increased** to `1.0.1` or `1.1`, since this would be the first PR made since `1.0`, but no coding changes occurred.

SamBumgardner commented:
Since this doesn't alter the project in any way, I don't think the version number should be incremented. We can change it later if we need to. 

I do prefer having the version number on its own line, because it's a separate subject from the lines before it. The rest of the changes look good to me as well, so I'll merge this in now, unless you have any objections.

NBumgardner commented:
Thanks for the feedback. It is ready to be merged.

On Wed, Sep 21, 2016 at 5:52 PM, Sam Bumgardner notifications@github.com
wrote:

> Since this doesn't alter the project in any way, I don't think the version
> number should be incremented. We can change it later if we need to.
> 
> I do prefer having the version number on its own line, because it's a
> separate subject from the lines before it. The rest of the changes look
> good to me as well, so I'll merge this in now, unless you have any
> objections.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> https://github.com/SamBumgardner/Floe/pull/1#issuecomment-248765927, or mute
> the thread
> https://github.com/notifications/unsubscribe-auth/ALS5TjIrxdLrOn9kvY4C54d_q_0IkqHbks5qsbUhgaJpZM4KDFPf
> .
